### PR TITLE
 animations will work onClick, missing several tasks on roadmap

### PR DESCRIPTION
**Pull request by thegreatsantini**

### CODE REVIEW

**Comment by thegreatsantini** [comment]
Tasks
1) fix animation functionality to work on keyup
2) alert when crossed finish line
3) positioning on checkered flag and carts (with flexbox)
4) activate keyup only after timer has started
5) I want to learn how to use flexbox to animate the carts and better positioning (see #3)

**Comment by thegreatsantini** [comment]
Tasks
1) ~~fix animation functionality to work on keyup~~
2) alert when crossed finish line
3) positioning on checkered flag and carts (with flexbox)
4) activate keyup only after timer has started
5) I want to learn how to use flexbox to animate the carts and better positioning (see #3)

File: app.js [hunk 1]
```diff
@@ -19,24 +6,52 @@ window.onload = function() {
   var buttonStart = document.getElementById("button-start");
   var buttonStop = document.getElementById("button-stop");
   var buttonReset = document.getElementById("button-reset");
+  var cart1 = document.querySelector('#cart1') 
   var Interval;
+  let cart1Left = 0;
```
Comment (brandiw) [inline app.js]: Remove commented out code from file. (This code is preserved by your git commits, you can always go back for it later if you need it so no need to keep it in comments)

File: app.js [hunk 2]
```diff
@@ -6,52 +6,49 @@ window.onload = function() {
...
+    } else if ( e.keyCode === 39 || e.keyCode === 37 ) {
+      cart2Left += 25;
+    cart2.style.left = cart2Left + "px";
+    }
+  };
+  document.addEventListener("keyup", moveCart);
 
   buttonStart.onclick = function() {
     clearInterval(Interval);
     Interval = setInterval(startTimer, 10);
```
Comment (brandiw) [inline app.js]: Time is in milliseconds! 1000ms === 1s.

File: app.js [hunk 3]
```diff
@@ -6,52 +6,49 @@ window.onload = function() {
...
-//   };
+  //   buttonReset.onclick = function() {
+  //     clearInterval(Interval);
+  //     tens = "00";
+  //     seconds = "00";
+  //     appendTens.innerHTML = tens;
+  //     appendSeconds.innerHTML = seconds;
+  //   };
 
   function startTimer() {
```
Comment (brandiw) [inline app.js]: Where is the check for win condition? What stops this timer?

**Review by brandiw — COMMENTED** [review]
LMK any lingering questions!

**Pull request closed without merging**